src/pages/home/sidebar/FloatingActionButtonAndPopover.tsx

```diff
@@ -137,21 +137,17 @@ function FloatingActionButtonAndPopover({onHideCreateMenu, onShowCreateMenu, isT
         canBeMissing: true,
     });
     const [tryNewDot] = useOnyx(ONYXKEYS.NVP_TRY_NEW_DOT, {selector: tryNewDotOnyxSelector, canBeMissing: true});
```

Comment:
Why do we need two versions of `ONYXKEYS.NVP_TRY_NEW_DOT`value? One with `tryNewDotOnyxSelector` selector, and one without

Reply:
Should we instead add the missing data to `tryNewDotOnyxSelector`?

Reply:
@TaduJR, gentle bump on the above